modules/ROOT/pages/remove-client-app-latest-task.adoc

```diff
@@ -1,12 +1,28 @@
-= Removing API instance Contracts on API Manager
+= Revoke and Delete API Contracts
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-As Organization Administrator of the master organization, to delete an application that you gave access to your API:
+As the Organization Administrator of the master organization, you can revoke or delete a contract granting an application access to your API. When you revoke a contract, you can restore the contract at any time. 
 
-. Click API Manager > Contracts.
-. Select the name of the application to delete.
-. Click Delete image:trash.png[trash icon].
+Deleting a contract deletes the approved contract from Anypoint Exchange. Before you delete a contract, you must first revoke it. 
 
-This action deletes any granted access to that contract from the API Exchange portal.
\ No newline at end of file
+To delete a contract with an application:
+
+. In API Manager, click *API Administration*.
+. Click the name of the API instance for which you want to revoke access.
+. Click the `<api version>:<api instance>` link, for example, `v1:16046365`. 
```

Comment:
Same as above. If the API Instance has a Label, then the label is shown.

Reply:
Done! Thanks!